load/UpdateDBPAccessTable.py

```diff
@@ -15,7 +15,10 @@ def __init__(self, config, db, dbOut, languageReader):
 		self.db = db
 		self.dbOut = dbOut
 		self.languageReader = languageReader
+		self.orgWithGeneralDownloadPermission = self.config.org_with_general_download_permission if self.config.org_with_general_download_permission != None else {'SIL', 'SIL MSEA'}
 
+	def hasGeneralDownloadPermission(self, record):
```

Comment:
rename to hasGrantedGeneralDownloadPermission

Reply:
@bradflood I have renamed it as you can see in the following line:

https://github.com/faithcomesbyhearing/dbp-etl/pull/117/files#diff-1fd781079e6b9caf6a8d8693ceb7aeb9b93a55e172acd2406273d04f4c1a5c1bR20